src/GitVersionTask/GetVersion.cs

```diff
@@ -1,108 +1,129 @@
 namespace GitVersionTask
 {
-    using Microsoft.Build.Framework;
+    using System;
+    using GitVersion;
 
-    public class GetVersion : GitVersionTaskBase
+    public static class GetVersion
```

Comment:
@stazz, why the change to `static` here?

Reply:
Just like `GenerateGitVersionInformation` (and one other task too), this class has no state on its own anymore, so no instances of it will be created either.